tests/utils/test_deprecated_kwargs.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from janitor.utils import deprecated_kwargs
+
+
+@pytest.mark.utils
+@pytest.mark.parametrize(
+    "arguments, kwargs",
+    [
+        (["a"], dict(a=1)),
+        (["b"], dict(b=2)),
+        (["a", "b"], dict(a=1, b=2)),
+        (["b", "a"], dict(a=1, b=2)),
+    ],
+)
+def test_arguments(arguments, kwargs):
+    @deprecated_kwargs(*arguments)
+    def simple_sum(alpha, beta, a=0, b=0):
+        return alpha + beta
+
+    with pytest.raises(ValueError):
+        simple_sum(1, 2, **kwargs)
+
+
+@pytest.mark.utils
+@pytest.mark.parametrize(
+    "arguments, args, excepted",
```

Comment:
1. I suppose you mean `expected` here.
2. Can you change the name of `args` to `func_args`? It makes it a little clearer that you're trying to parametrize the args of the `simple_sum` function rather than the decorator args.